docs/documentation/index.html

```diff
@@ -3139,6 +3145,33 @@ <h3>Message Types</h3>
 					</table>
 
 					<div class="row" style="height:40px;"></div>
+
+					<h1>Frequently Asked Questions</h1>
+
+					<div class="panel panel-default">
+						<div class="panel-heading">
+							<h3 class="panel-title">Can I use Sourcetrail on confidental source code?</h3>
+						</div>
+						<div class="panel-body">
+							Yes. Sourcetrail works completely offline and establishes no connection to the internet whatsoever. All data stays on your computer.
```

Comment:
This is not entirely true. Sourcetrail runs an update check that asks our server if a more recent version is available. But that can be disabled in the preferences. If disabled, Sourcetrail establishes no connection to the internet whatsoever ;)

Maybe just write that Sourcetrail keeps the data completely offline and (if not disabled) only establishes a connection to the internet to check if a more recent version is available.

Reply:
Changed to what you wanted to replace the current answer with. However, I think it may be a good idea to mention this in the overview section at some point.